ts/input/tex/physics/PhysicsMappings.ts

```diff
@@ -127,7 +127,8 @@ new CommandMap('Physics-expressions-macros', {
   'Trace':            ['Expression', false, 'Tr'],
   'rank':             'NamedFn',
   'erf':              ['Expression', false],
-  'Res':              ['OperatorApplication', '{\\rm Res}', '(', '[', '{'],
+  'Residue':          ['Macro', '{\\rm Res}'],
```

Comment:
I'm wondering if you want this to be `\mathrm{Res}` now that `\mathrm{}` will produce a single `<mi mathvarian="normal">Res</mi>` rather than a separate `mi` for each letter?

Reply:
Done and likewise for `Re` and `Im`.